vue-todo-list/src/components/todoFilter.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <li>
+    <a
+      v-bind:class="filter.class"
+      v-bind:href="filter.link"
+      v-on:click="changeFilter"
```

Comment:
알고 있으실거라 생각하지만

```suggestion
      @click="changeFilter"
```
으로 표현이 가능합니다

Reply:
스타일 가이드를 보면 축약을 하거나, 혹은 축약을 하지 않거나 둘 중에 하나만 유지하면 된다고 하더라고요! 저는 우테코에서의 축약하지 말아라. 라는 것을 생각해 축약을 사용하지 않았습니다! 아래의 링크를 참고해주세요~
https://kr.vuejs.org/v2/style-guide/#%EC%B6%95%EC%95%BD%ED%98%95-%EB%94%94%EB%A0%89%ED%8B%B0%EB%B8%8C-%EB%A7%A4%EC%9A%B0-%EC%B6%94%EC%B2%9C%ED%95%A8